shared/src/packages/users/libs/validation-schemas/enums/user-validation-rule.enum.ts

```diff
@@ -1,7 +1,7 @@
 const UserValidationRule = {
   PASSWORD: /^(?=.*[A-Za-z])(?=.*\d)[\dA-Za-z]{6,20}$/,
-  NAME: /^[A-Za-z][\s'A-Za-z-]{0,39}$/,
-  PHONE: /^\+\d{8,19}$/,
+  NAME: /^['A-Za-z-]{1,40}$/,
```

Comment:
This will eliminate check for first symbol to be letter

Reply:
Why would we do that? In that case, `-` and `'` would be considered a valid name.